src/static/stylus/base_template.styl

```diff
@@ -73,6 +73,16 @@
 .hidden.menu
     display none
 
+.progress-style
+    margin 0
+    height 0
+    -ms-flex 1 0 auto
+    flex 1 0 auto
+    overflow hidden
+
+.progress24height
```

Comment:
instead just do it like so at the bottom of `management.tag`, but in regular css, also I don't think we need a new class?

```
.progress
     margin 0
     height 0
     -ms-flex 1 0 auto
     flex 1 0 auto
     overflow hidden
    .bar
        height 24px
```

In regular CSS this owuld be something like:

```
.progress {
    yadda yadda
}
.progress .bar {
    height: 24px;
}
```

Reply:
This has been moved